stories/Portals.stories.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Stories for Portal components.
+ *
+ * @flow
+ */
+
+
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import MeetingControlBar from 'universal/modules/meeting/components/MeetingControlBar/MeetingControlBar';
+import MeetingHelpDialog from 'universal/modules/meeting/components/MeetingHelpDialog/MeetingHelpDialog';
+import phaseHelpLookup from 'universal/utils/meetings/helpLookups';
+
+import RetroBackground from './components/RetroBackground';
+import StoryContainer from './components/StoryContainer';
+
+const EXAMPLE_PHASE = 'firstcall';
+
+storiesOf('Loadable Help Dialog', module)
+  .add('toggle the help dialog', () => (
+    <RetroBackground>
+      <StoryContainer
+        description={`This demonstrates a help dialog (available for all teammates)
+          and an optional tip (for the facilitator in the control bar)`}
+        render={() => (
+          <div style={{padding: '50vh 0 0'}}>
+            <div style={{display: 'flex', justifyContent: 'flex-end', padding: '0 1.25rem 1.25rem 0'}}>
+              <MeetingHelpDialog phase={EXAMPLE_PHASE} />
+            </div>
+            <MeetingControlBar>
+              {phaseHelpLookup[EXAMPLE_PHASE].facilitatorBarTip}
```

Comment:
my only concern here is when we show these.
A few of these should probably only be triggered on an action, eg the voting phase requirements. 
then again, i think the conditional requirements is out of scope of this PR, so i think it's good to start here!

Reply:
Agreed! When we start using this I think I may just hide some of them until we can make them conditional. I’ll consider as the criteria for an issue to wire up.